openstack/networking/v2/extensions/fwaas_v2/rules/results.go

```diff
@@ -0,0 +1,42 @@
+package rules
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+// Rule represents a firewall rule
+type Rule struct {
+	ID                   string `json:"id"`
+	Name                 string `json:"name,omitempty"`
+	Description          string `json:"description,omitempty"`
+	Protocol             string `json:"protocol"`
+	Action               string `json:"action"`
+	IPVersion            int    `json:"ip_version,omitempty"`
+	SourceIPAddress      string `json:"source_ip_address,omitempty"`
+	DestinationIPAddress string `json:"destination_ip_address,omitempty"`
+	SourcePort           string `json:"source_port,omitempty"`
+	DestinationPort      string `json:"destination_port,omitempty"`
+	Shared               bool   `json:"shared,omitempty"`
+	Enabled              bool   `json:"enabled,omitempty"`
+	PolicyID             string `json:"firewall_policy_id"`
+	Position             int    `json:"position"`
```

Comment:
I'm unable to tell how this field is added to the result here: https://github.com/openstack/neutron-fwaas/blob/27906d0acff349c9e5ac7955d203fb2233c295db/neutron_fwaas/db/firewall/v2/firewall_db_v2.py#L269

Can you verify this field is really part of the JSON result?

Reply:
Well spotted! I've verified that it's not. Removed in commit f1ad97effda2c92edd871a4728da1ce7c2f22479

Reply:
The JSON result is also returning project_id, so I've added this in commit be00d44db04b8dd9fb72ed3acb2a66687f39f4c6